### DISCUSSION TITLE
Issue #1253 workaround

### DISCUSSION THREAD

**Discussion by AntnPrcl** [FlareSolverr/FlareSolverr#1295] [Show and tell]
Hi,
I didn't want to open a duplicate issue on this subject, so i'm writing here as the discussion on #1253 are blocked.
I found myself yesterday on the same situation as described in #1291 as:
Flaresolverr running on a docker in v.3.3.20 with the flaresolverr_service.py updated as stated in the comment of #1253. The modification of the .py file was sucessful during a while, but stopped working few days ago with a 500: Internal server error. I succesfully make it work again this morning with this : 
* Deletion of the docker `docker stop flaresolverr && docker rm flaresolverr`
* Manually change the docker-compose.yml file to pull the v.3.3.21 version
* `docker exec -it flaresolverr sh` to open a shell into the docker
* Replacement of the flaresolverr_service.py by the same one shared on #1253 (https://github.com/MCG-pok/FlareSolverr/blob/master/src/flaresolverr_service.py)
* Restart of the docker
* Enjoy

During the tests i've tried removing the v3.3.20 docker and replace it by the v3.3.21 without changing the .py file and i was still getting the 500 error, so I don't think it's just about a restart or a recreation of the docker...

Hope it helps !

**Comment by dreadwrr** [comment]
im getting this issue currently on windows 11 challenge detected but then fails and doesnt tell you why. Says error 500
